src/initialization.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+const distance = require('ml-distance-euclidean');
+
+/**
+ * Choose K different random points from the original data
+ * @ignore
+ * @param {Array<Array<Number>>} data - Points in the format to cluster [x,y,z,...]
+ * @param {Number} K - Number of clusters
+ * @return {Array<Array<Number>>} - Initial random points
+ */
+function random(data, K) {
```

Comment:
This probably belongs outside of this project. Or maybe there is already something on npm for that ?